password/test/tasty/Scrypt.hs

```diff
@@ -30,9 +31,7 @@ testScrypt = testGroup "scrypt"
                  checkPassword
                  extractParams
                  testsParams8Rounds
-#ifndef IS_MAC_OS
   , testProperty "scrypt <-> cryptonite" $ withMaxSuccess 10 checkScrypt
```

Comment:
I don't feel like acting we're testing something when we're not is a good idea.

I'd prefer:
```hs
#ifndef darwin_HOST_OS
    , testProperty "scrypt <-> cryptonite" $ withMaxSuccess 10 checkScrypt
#endif
```

And just not defining `checkScrypt`.

Reply:
That's a good point.  I've changed this back in b5a96ef.